x-pack/plugins/observability_solution/synthetics/server/routes/settings/params/delete_param.ts

```diff
@@ -13,25 +13,45 @@ import { DeleteParamsResponse } from '../../../../common/runtime_types';
 
 export const deleteSyntheticsParamsRoute: SyntheticsRestApiRouteFactory<
   DeleteParamsResponse[],
-  unknown,
+  { id?: string },
   unknown,
   { ids: string[] }
 > = () => ({
   method: 'DELETE',
-  path: SYNTHETICS_API_URLS.PARAMS,
+  path: SYNTHETICS_API_URLS.PARAMS + '/{id?}',
   validate: {},
   validation: {
     request: {
-      body: schema.object({
-        ids: schema.arrayOf(schema.string()),
+      body: schema.nullable(
+        schema.object({
+          ids: schema.arrayOf(schema.string()),
+        })
+      ),
+      params: schema.object({
+        id: schema.maybe(schema.string()),
       }),
     },
   },
-  handler: async ({ savedObjectsClient, request }) => {
-    const { ids } = request.body;
+  handler: async ({ savedObjectsClient, request, response }) => {
+    const { ids } = request.body ?? {};
+    const { id: paramId } = request.params ?? {};
+
+    if (ids && paramId) {
+      return response.badRequest({
+        body: `Both param id  and body parameters cannot be provided`,
```

Comment:
```suggestion
        body: `Both param id and body parameters cannot be provided`,
```
i18n if this is an error message intended for the user.